doobie/src/it/scala/za/co/absa/fadb/doobiedb/DoobieTest.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.fadb.doobiedb
+
+import cats.effect.IO
+import doobie.util.log.{LogEvent, LogHandler}
+import doobie.util.transactor.Transactor
+import za.co.absa.fadb.DBSchema
+
+trait DoobieTest {
+  case class Actor(actorId: Int, firstName: String, lastName: String)
+  case class GetActorsQueryParameters(firstName: Option[String], lastName: Option[String])
+  case class CreateActorRequestBody(firstName: String, lastName: String)
+
+  import za.co.absa.fadb.naming.implementations.SnakeCaseNaming.Implicits._
+  object Runs extends DBSchema
+
+  val printSqlLogHandler: LogHandler[IO] = new LogHandler[IO] {
+    def run(logEvent: LogEvent): IO[Unit] =
+      IO {
+        println(logEvent.sql)
+      }
+  }
```

Comment:
```suggestion
  val printSqlLogHandler: LogHandler[IO] = (logEvent: LogEvent) => IO {
    println(logEvent.sql)
  }
```

❓

Reply:
ok refactored in this fashion as one-liner

Reply:
After downgrade of Doobie version this is no longer relevant. The support for LogHandler is not available in the downgraded version.